packages/react-native-worklets/android/src/main/java/com/swmansion/worklets/AnimationFrameQueue/AnimationFrameCallback.java

```diff
@@ -1,4 +1,4 @@
-package com.swmansion.worklets;
+package com.swmansion.worklets.AnimationFrameQueue;
```

Comment:
Can we please avoid using `com.swmansion.worklets.AnimationFrameQueue` package in favor of `com.swmansion.worklets`? There could be a name conflict with `AnimationFrameQueue` from `com.swmansion.reanimated` package (somehow), if that's the case please resolve the conflict or rename one of them.